requirements - Copy.txt

```diff
@@ -0,0 +1,11 @@
+kubernetes==11.0.0
```

Comment:
Why does this exist?

Reply:
again, used during development, it needed to be deleted